.zappr-example.yml

```diff
@@ -1,8 +1,10 @@
 approvals:
-  # PR needs at least 4 approvals
-  minimum: 4
+  # PR needs at least 2 approvals
+  minimum: 2
```

Comment:
Maybe we could also make the "users" example slightly more generic. Currently we have this:

```yaml
# this guy
users:
  - mfellner
```